RELEASE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Cumulative Release Notes for rosette-common-java-api
 
+## 37.1.1
```

Comment:
We're a bit more careful (sometimes, at least) about semver here - I think this should really be 37.2.0 and not 37.1.1 in that case

Reply:
per semver.org, updated backwards-compatible functionality deserves a minor release; point releases are only meant for bug fixes.  We definitely don't always stick to this, but that't what led to the previous release being 37.1.0 and not 37.0.5